portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,6 +44,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String comment = getParameter(request, "comment-input", "");
+    if (!comment.isBlank()) { 
```

Comment:
Looks like we can simplify a bit further now. In line 50, we don't need `getParameter`, we can just call `request.getParameter("comment-input")` and then here in line 51 check `if (comment != null)`.

Reply:
I tried that but it didn't work. Apparently it's never null for strings.

Reply:
I see what your saying. although, text sections apparently never return null. so I had to use the `isBlank()` function. `isBlank` also checks for spaces too so that's a bonus.